client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -31,23 +32,39 @@ export function PatternPreview( {
 	}
 
 	return (
-		<div
-			className={ classNames( 'pattern-preview', {
-				'pattern-preview_category-gallery': isCategoryPreview,
-				'is-loading': ! renderedPattern,
-			} ) }
+		<ResizableBox
+			enable={ {
+				top: false,
+				right: true,
+				bottom: false,
+				left: false,
+				topRight: false,
+				bottomRight: false,
+				bottomLeft: false,
+				topLeft: false,
+			} }
+			handleWrapperClass={'pattern-preview__resizer'}
+			minWidth={500}
```

Comment:
```suggestion
			minWidth={375}
```
This is closer to a modern mobile viewport (iPhone 15 has a 393px wide viewport).